### PR TITLE
Hide waiver as a payment method at checkin

### PR DESCRIPTION
**Pull request by jecihjoy**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
Screenshot from 2024-01-23 16-55-58

Hide the waiver option at checkin since it's not applicable at checkin
*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by makombe** [comment]
@jecihjoy Do we still need this?

File: packages/esm-billing-app/src/billing-form/visit-attributes/visit-attributes-form.component.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ const VisitAttributesForm: React.FC<VisitAttributesFormProps> = ({ setAttributes
       />
     );
   }
+  paymentModes = paymentModes?.filter((p) => p.uuid !== 'eb6173cb-9678-4614-bbe1-0ccf7ed9d1d4');
```
Comment (makombe) [inline packages/esm-billing-app/src/billing-form/visit-attributes/visit-attributes-form.component.tsx:80]: I would prefer we move this to the `usePaymentMethods` hook

**Comment by ojwanganto** [comment]
I think you shouldn't see waiver as an option at check in.

File: packages/esm-billing-app/src/billing-form/visit-attributes/visit-attributes-form.component.tsx (hunk 1 above)
Comment (ojwanganto) [reply packages/esm-billing-app/src/billing-form/visit-attributes/visit-attributes-form.component.tsx:80]: We can also move the UUID to the constants file I guess

Comment (jecihjoy) [reply packages/esm-billing-app/src/billing-form/visit-attributes/visit-attributes-form.component.tsx:80]: This was merged in through a separate PR, I will add the logic inside the hook.
I'll check what Donald has on the release packages so we can setup different payments as part of the packages

**Comment by jecihjoy** [comment]
Merged in a separate PR

**Pull request closed without merging**